src/DearImGui.hs

```diff
@@ -293,7 +293,7 @@ sameLine = liftIO do
 text :: MonadIO m => String -> m ()
 text t = liftIO do
   withCString t \textPtr ->
-    [C.exp| void { Text($(char* textPtr)) } |]
+    [C.exp| void { Text("%s", $(char* textPtr)) } |]
```

Comment:
Is this change necessary? Maybe we can do it in a separate PR?

Reply:
This triggers a security error that prevents me from compiling on darwin, so, for the moment, yes it's necessary.  I can revert before merging if you like.

Reply:
Oh, wow! Sounds like we should bring this fix in, but could we do it in a separate PR?

Reply:
done in #6